.npmrc

```diff
@@ -0,0 +1 @@
+@OWNER:registry=https://npm.pkg.github.com
```

Comment:
what does `@OWNER` do? i've only published this package to npm, so i'm not sure if this is needed